integrations/acquisition/covidcast/test_is_latest_issue.py

```diff
@@ -0,0 +1,262 @@
+"""Integration tests for covidcast's is_latest_issue boolean."""
+# standard library
+import unittest
+import time
+import threading
+
+
+# third party
+from aiohttp.client_exceptions import ClientResponseError
+import mysql.connector
+import pytest
+from prettytable import PrettyTable, from_db_cursor
+    
+# first party
+from delphi.epidata.acquisition.covidcast.logger import get_structured_logger
+from delphi_utils import Nans
+from delphi.epidata.client.delphi_epidata import Epidata
+from delphi.epidata.acquisition.covidcast.database import Database, CovidcastRow
+from delphi.epidata.acquisition.covidcast.covidcast_meta_cache_updater import main as update_covidcast_meta_cache
+import delphi.operations.secrets as secrets
+
+# py3tester coverage target (equivalent to `import *`)
+#
+__test_target__ = 'delphi.epidata.acquisition.covidcast.database'
+
+nmv = Nans.NOT_MISSING.value
+class CovidcastLatestIssueTests(unittest.TestCase):
+  """Tests covidcast is_latest_issue caching."""
+
+  
+  def setUp(self):
+    """Perform per-test setup."""
+
+    # connect to the `epidata` database
+    cnx = mysql.connector.connect(
+        user='user',
+        password='pass',
+        host='delphi_database_epidata',
+        database='covid')
+    cur = cnx.cursor()
+
+    # clear all tables
+    cur.execute("truncate table signal_load")
+    cur.execute("truncate table signal_history")
+    cur.execute("truncate table signal_latest")
+    cur.execute("truncate table geo_dim")
+    cur.execute("truncate table signal_dim")
+    # reset the `covidcast_meta_cache` table (it should always have one row)
+    cur.execute('update covidcast_meta_cache set timestamp = 0, epidata = "[]"')
+    
+    cnx.commit()
+    cur.close()
+
+    # make connection and cursor available to the Database object
+    self._db = Database()
+    self._db._connection = cnx
+    self._db._cursor = cnx.cursor()
+
+    # use the local instance of the Epidata API
+    BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
+
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
+
+    #Commonly used SQL commands:
+    self.viewSignalLatest = '''SELECT * FROM `signal_latest`'''
+    self.viewSignalHistory = '''SELECT * FROM `signal_history`'''
+    self.viewSignalDim = '''SELECT `source`, `signal` FROM `signal_dim`'''
+    self.viewGeoDim = '''SELECT `geo_type`,`geo_value` FROM `geo_dim`'''
```

Comment:
For easier maintainability, use e.g. `Database.history_table` instead of "signal_history" in these